lib/fine-types/fine-types.cabal

```diff
@@ -89,14 +91,17 @@ test-suite unit
     , containers
     , deepseq
     , fine-types
+    , lens
```

Comment:
```suggestion
```

Thanks for the changes! 😊 It looks like the package dependency can be removed as well? 🤔